ddtrace/appsec/internal/events/attack.py

```diff
@@ -0,0 +1,49 @@
+from typing import List
+from typing import Union
+
+import attr
+
+from ddtrace.appsec.internal.events.context import Context_0_1_0
+
+
```

Comment:
Should there be a `Context = Union[Context_0_1_0]` too?

Reply:
It makes sense for the `Attack` struct because there will be more event types in the future but the context might only get new versions.